authenticate/handlers.go

```diff
@@ -179,11 +168,6 @@ func (a *Authenticate) SignIn(w http.ResponseWriter, r *http.Request) error {
 	options := a.options.Load()
 	state := a.state.Load()
 
-	sharedCipher, err := cryptutil.NewAEADCipherFromBase64(options.SharedKey)
```

Comment:
This was... probably very expensive to do on a per request basis and could ... maybe even exhaust the entropy source? IDK.